dbt_common/contracts/config/base.py

```diff
@@ -150,30 +172,6 @@ def _merge_dicts(cls, src: Dict[str, Any], data: Dict[str, Any]) -> Dict[str, An
             )
         return result
 
-    def update_from(
```

Comment:
there don't appear to be explicit usages of `update_from` in dbt-adapters: https://github.com/search?q=repo%3Adbt-labs%2Fdbt-adapters%20update_from&type=code

Reply:
The signature change here looks like it will have a very minor impact (just to dbt-core). We could release a dbt-common major version bump for this change since it is technically breaking, but that might be overkill given its low usage. At least a minor bump makes sense to me though.